src/hooks/useRecordTypeLabel/useRecordTypeLabel.tsx

```diff
@@ -0,0 +1,8 @@
+import {useRecordTypes} from '../useRecordTypes';
+
+
+export const useRecordTypeLabel = (targetID: string = '') => {
+  const { recordTypes = [] } = useRecordTypes();
+
+  return recordTypes.find(({id}) => id === targetID)?.label || '';
```

Comment:
I don't know why, but this little one-liner makes me smile. There's a ton happening in this one line (finding the type with the matching ID, elvising our way to its label, and providing a default), but it's still clear what's going on 🙂 I love typescript sometimes...